src/plugins/pronoundb/PronounComponent.tsx

```diff
@@ -0,0 +1,23 @@
+import { fetchPronouns } from "./utils";
+import { useAwaiter } from "../../utils/misc";
+import { MessageHeaderProps, PronounMapping } from "./types";
+import { findByProps } from "../../webpack";
+import { proxyLazy } from "../../utils";
+
+const styles: Record<string, string> = proxyLazy(() => findByProps("timestampInline"));
+
+export default function PronounComponent({ message }: MessageHeaderProps) {
+    // Don't bother fetching bot or system users
+    if (message.author.bot && message.author.system) return null;
+
+    const [result, error, isPending] = useAwaiter(() => fetchPronouns(message.author.id));
+
+    // If the fetching completed successfully and the result was not "unspecified", then return a span with the pronouns
+    if (!isPending && result && result !== "unspecified") return (
+        <span className={`${styles.timestampInline} ${styles.timestamp}`}>• {PronounMapping[result]}</span>
```

Comment:
```suggestion
        <span className={`${styles.timestampInline} ${styles.timestamp}`}>• {PronounMapping[result] ?? result}</span>
```

shouldn't happen but just in case, noone wants to see undefined

Reply:
no one also wants to see two letters
wouldn't just not showing anything if there is no mapping work better?

Reply:
i guess so